docs/actions.md

```diff
@@ -0,0 +1,10 @@
+# Actions that can be redefined by the user
```

Comment:
The table is excellent: but we need much more docs:

* intro: what it is and what problem it solves
* how it can be configured
* real examples

Reply:
I've tried to fix that in ef5b76a47cec010f0d0c1f92260a234e5bfab998